frontend/src/views/Lobby.tsx

```diff
@@ -58,9 +58,17 @@ function LobbyPage() {
     setDifficulty(room.difficulty);
   };
 
-  const deleteUser = (user: User) => {
-    // Make rest call to delete user from room
-    console.log(user);
+  const kickUser = (userId: string) => {
+    removeUser(currentRoomId, {
+      initiatorId: currentUser!.userId as string,
+      userId,
+    })
+      .then(() => {
+        setLoading(true);
+      })
```

Comment:
This setup will set loading to true once the operation completes, and never turn it off? I believe what you want is to set loading to true _right before the operation begins_ (at the top of `kickUser(...)`), and then set loading to false in the `then(...)` block.